README.md

```diff
@@ -54,81 +54,29 @@ straightforward.
 
 ## Usage
 
-WORKSPACE
+`lucidsoftware/rules_scala` isn't on the [Bazel Central Registry](https://registry.bazel.build/), so
+you'll need to pull it in via `archive_override`. Be sure to replace `<COMMIT>` with the
+latest commit on `lucid-master` and `<INTEGRITY>` with the hash suggested by Bazel after the
+dependency is first loaded.
 
-```python
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+MODULE.bazel
 
-# rules_java
-http_archive(
-    name = "rules_java",
-    sha256 = "a9690bc00c538246880d5c83c233e4deb83fe885f54c21bb445eb8116a180b83",
-    url = "https://github.com/bazelbuild/rules_java/releases/download/7.12.2/rules_java-7.12.2.tar.gz",
-)
+```starlark
+bazel_dep(name = "rules_scala_annex")
 
-# Load rules_scala_annex
-rules_scala_annex_version = "lucid_2024-11-18"
+rules_scala_annex_version = "<COMMIT>"
 
-http_archive(
-    name = "rules_scala_annex",
-    integrity = "sha256-WjZvojiclkiyVxQ1NqkH1lDeGaDLyzQOGiDsCfhVAec=",
+archive_override(
```

Comment:
Do folks need to set up toolchains in their repos with bzlmod or are our toolchains just set up in their repo by including our ruleset?

Reply:
Nope, they're set up by including our ruleset. I think that's the one aspect of external repositories in Bzlmod that isn't isolated—toolchains are shared throughout the entire Bazel server. See the [documentation](https://bazel.build/extending/toolchains#toolchain-resolution) on toolchain resolution, namely the bit about the order in which toolchains are resolved.

Reply:
Nice.